lldb/test/API/lang/swift/regex/TestSwiftRegex.py

```diff
@@ -0,0 +1,45 @@
+# TestSwiftRegex.py
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+# ------------------------------------------------------------------------------
```

Comment:
We no longer put this header into tests, swift-lldb and llvm.org LLDB are now both under Apache license, so this is implicit.